src/Helper/MmenuHelper.php

```diff
@@ -119,6 +119,7 @@ public static function processModuleSettings(Module $module, string $jsTemplateN
         $jsTemplate->configuration = $configuration;
 
         // Add module JavaScript (and replace insert tags, see #66)
-        $GLOBALS['TL_BODY'][] = Controller::replaceInsertTags($jsTemplate->parse());
+        //Revert to the original without inserttags
+        $GLOBALS['TL_BODY'][] = $jsTemplate->parse();
```

Comment:
This was a user requirement from #66 - https://github.com/dklemmt/contao_dk_mmenu/pull/66#issuecomment-1345256801

Reply:
Yes I know, but at the moment I have no idea how to implement the  newinserttagparser service. That would have to be done by someone with more programming experience ;-)